# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../csprng/0003-utils-fix-csprng-generate-to-include-limits.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From f2af70814524601ef2c5b2bf347ee7751c062efd Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Thu, 5 Sep 2013 00:36:29 +0200
Subject: [PATCH] utils: fix csprng-generate to include limits.h

---
 utils/csprng-generate.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/utils/csprng-generate.c b/utils/csprng-generate.c
index 9d6fa58..c34b122 100644
--- a/utils/csprng-generate.c
+++ b/utils/csprng-generate.c
@@ -53,6 +53,7 @@ along with CSRNG.  If not, see <http://www.gnu.org/licenses/>.
 
 #include <stdlib.h>
 #include <stdio.h>
+#include <limits.h> /* for INT_MAX, LONG_MAX */
 
 #include <csprng/csprng.h>
 #include <csprng/helper_utils.h>
-- 
1.7.2.3

